nanoFramework.Tools.DebugLibrary.Shared/WireProtocol/Commands.cs

```diff
@@ -528,6 +551,103 @@ public override bool PrepareForSend(byte[] data, int length, int offset = 0)
                 return true;
             }
         }
+       
+        /// <summary>
+        /// Perform storage operation on the target device.
+        /// </summary>
+        public class Monitor_StorageOperation : OverheadBase
+        {
+            public uint Operation = (byte)StorageOperation.None;
+            public uint NameLength = 0;
+            public uint DataLength = 0;
+            public byte[] Data = new byte[0];
+
+            public class Reply
+            {
+                public uint ErrorCode;
+            };
+
+            public void PrepareForSend(
+              StorageOperation operation,
+              string name)
+            {
+                Operation = (byte)operation;
+                Data = Encoding.UTF8.GetBytes(name);
+                NameLength = (uint)Data.Length;
+            }
+
+            public void PrepareForSend(
+                StorageOperation operation,
+                string name,
+                byte[] data,
+                int offset,
+                int length)
+            {
+                Operation = (byte)operation;
```

Comment:
We should avoid repeating the logic here, in case it changes. 
Better replace these 3 lines of code with a call to `PrepareForSend(
              StorageOperation operation,
              string name)`